main/cicd/cicd-pipeline/config/infra/cloudformation.yml

```diff
@@ -199,6 +199,113 @@ Resources:
                 Effect: Allow
                 Resource: !Ref PipelineNotificationsTopic
 
+  PolicyAppDeployer:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: Allows CodePipeline to create resources required for deploying SWB
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Action:
+              - iam:DeleteRole
+              - iam:DeleteRolePolicy
+              - iam:DetachRolePolicy
+              - iam:GetRole
+              - iam:GetRolePolicy
+              - iam:*TagRole*
+            Resource: '*'
+            Effect: Allow
+          - Action:
+              - iam:PassRole
+            Effect: Allow
+            Resource: !Sub 'arn:aws:iam::${AWS::AccountId}:role/${self:custom.settings.namespace}-LaunchConstraint'
```

Comment:
So AppDeployer role is responsible for ensuring that ServiceCatalog is able to use this role?

Reply:
Yes, AppDeployer will be responsible for assigning launch constraint role to the newly created products in the out-of-the-box SC portfolio.